ide/che-core-ide-app/src/test/java/org/eclipse/che/ide/project/ProjectServiceClientTest.java

```diff
@@ -211,34 +207,34 @@ public void shouldEncodeUrlAndSearchResourceReferences() {
 
   @Test
   public void shouldCreateOneProjectByBatch() {
-    List<NewProjectConfigDto> configs = singletonList(prjConfig1);
-
-    client.createBatchProjects(configs);
-
-    verify(requestFactory).createPostRequest(anyString(), prjsArgCaptor.capture());
-    verify(asyncRequest).header(ACCEPT, MimeType.APPLICATION_JSON);
-    verify(loaderFactory).newLoader("Creating project...");
-    verify(asyncRequest).loader(messageLoader);
-    verify(asyncRequest).send(unmarshallablePrjsConf);
-    verify(unmarshaller).newListUnmarshaller(ProjectConfigDto.class);
-
-    assertEquals(1, prjsArgCaptor.getValue().size());
+    //    List<NewProjectConfigDto> configs = singletonList(prjConfig1);
+    //
+    //    client.createBatchProjects(configs);
+    //
+    //    verify(requestFactory).createPostRequest(anyString(), prjsArgCaptor.capture());
+    //    verify(asyncRequest).header(ACCEPT, MimeType.APPLICATION_JSON);
+    //    verify(loaderFactory).newLoader("Creating project...");
+    //    verify(asyncRequest).loader(messageLoader);
+    //    verify(asyncRequest).send(unmarshallablePrjsConf);
+    //    verify(unmarshaller).newListUnmarshaller(ProjectConfigDto.class);
+    //
+    //    assertEquals(1, prjsArgCaptor.getValue().size());
   }
 
   @Test
   public void shouldCreateFewProjectByBatch() {
-    List<NewProjectConfigDto> configs = Arrays.asList(prjConfig1, prjConfig2);
-
-    client.createBatchProjects(configs);
-
-    verify(requestFactory).createPostRequest(anyString(), prjsArgCaptor.capture());
-    verify(asyncRequest).header(ACCEPT, MimeType.APPLICATION_JSON);
-    verify(loaderFactory).newLoader("Creating the batch of projects...");
-    verify(asyncRequest).loader(messageLoader);
-    verify(asyncRequest).send(unmarshallablePrjsConf);
-    verify(unmarshaller).newListUnmarshaller(ProjectConfigDto.class);
-
-    assertEquals(2, prjsArgCaptor.getValue().size());
+    //    List<NewProjectConfigDto> configs = Arrays.asList(prjConfig1, prjConfig2);
+    //
+    //    client.createBatchProjects(configs);
```

Comment:
remove if not needed.

Reply:
This test is already reworked be one of my previous commits, in general I'm not sure if it is a good idea to review the PR that is in a work-in-progress state, I mean you will have many of your remarks outdated and useless. Please watch "status" label to be aware of current PR status not to waste your time.